app/views.py

```diff
@@ -1,19 +1,25 @@
 #!/usr/bin/python
-from flask import render_template, flash, session, redirect, request, Flask, url_for, jsonify
-from flask_oauthlib.client import OAuth
-from app import app, sql_session, login_manager, viewsClasses
-from app import models, viewsModel
-from .myfunctions import sortnumbers, getregressionline
+import datetime
```

Comment:
Using a utility like `isort` you can easily have standardized, readable imports.